src/Parser.jl

```diff
@@ -197,9 +199,9 @@ function parse_array(ps::ParserState, dictT::Type)
 end
 
 
-function parse_object(ps::ParserState, dictT::Type)
-    obj = dictT()
-    keyT = dictT.parameters[1]
+function parse_object(ps::ParserState, pc::ParserContext{DictType, <:Real}) where DictType
+    obj = DictType()
+    keyT = DictType.parameters[1]
```

Comment:
perhaps `keytype(...)`, although there's probably a reason we aren't already using it

Reply:
Yes, `keytype(...)` would be better.  I suspect the code was originally written before that function existed.